installer/dbscript/dbscript-2.sql

```diff
@@ -4417,3 +4417,20 @@ VALUES (1, 0, 1, 0, 0, @apiv2_attendance_configuration_data_group_id, @admin_rol
        (0, 1, 1, 0, 1, @apiv2_attendance_my_attendance_record_data_group_id, @ess_role_id),
        (0, 1, 1, 0, 0, @apiv2_attendance_employee_attendance_record_data_group_id, @admin_role_id),
        (0, 1, 1, 0, 0, @apiv2_attendance_employee_attendance_record_data_group_id, @supervisor_role_id);
+
```

Comment:
If I add a query to drop the comments column in Leave and LeaveRequest, will need to change `LeaveAPITest.yaml` also